plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/3dtiles/BuildingTransparency/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Slider } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+import { CSSProperties, useMemo } from "react";
+
+import { BaseFieldProps } from "../../types";
+
+import useHooks from "./hooks";
+
+const BuildingTransparency: React.FC<BaseFieldProps<"buildingTransparency">> = ({
+  value,
+  dataID,
+  editMode,
+  onUpdate,
+}) => {
+  const { options, handleUpdateNumber } = useHooks({
+    value,
+    dataID,
+    onUpdate,
+  });
+
+  const styleProps = useMemo(
+    () => ({
+      trackStyle: {
+        backgroundColor: "#00BEBE",
+      } as CSSProperties,
+      handleStyle: {
+        border: "2px solid #00BEBE",
+      } as CSSProperties,
+    }),
+    [],
+  );
```

Comment:
Same as another PR, can't you move this outside the component instead of using a useMemo?

Reply:
I fixed [fix: cull out styleProps](https://github.com/eukarya-inc/reearth-plateauview/pull/122/commits/eb57b2532e0a8b80d319fba76d6faef45840de51)